src/store/modules/user/actions.ts

```diff
@@ -45,8 +47,9 @@ const actions: ActionTree<UserState, RootState> = {
   async logout ({ commit }) {
     // TODO add any other tasks if need
     commit(types.USER_END_SESSION)
+    updateToken('')
+    updateInstanceUrl('')
```

Comment:
Can we have some method to reset the properties set?

Reply:
Added a method to clear data using a single method in oms package and will update the PR once it will be available in the oms release.